src/pages/home/report/ReportActionItem.js

```diff
@@ -185,12 +185,13 @@ ReportActionItem.defaultProps = defaultProps;
 
 export default compose(
     withWindowDimensions,
-    withOnyx({
-        draftMessage: {
-            key: ({
-                reportID,
-                action,
-            }) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS}${reportID}_${action.reportActionID}`,
+    withReportActionsDrafts({
+        propName: 'draftMessage',
+        transformValue: (drafts, props) => {
+            const {reportID, action} = props;
+            const draftKey = `${ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS}${reportID}_${action.reportActionID}`;
+            const draft = drafts[draftKey];
```

Comment:
Is it possible for `drafts` to be `undefined` or `null` ?
Maybe it's not possible since it's a collection value and would always be at least an empty object?

Reply:
I think no, it must be an empty object at least

https://github.com/Expensify/react-native-onyx/blob/d73900b7cb7bf82bed77cb6b6baabf8fe2eb3a0e/lib/Onyx.js#L398-L403

Reply:
Will add some protection to be on the safe side if this changes in the future.